tlatools/org.lamport.tlatools/src/tla2sany/drivers/SANY.java

```diff
@@ -408,6 +422,11 @@ private static void printUsage()
    * ExternalModuleTable.
    */
   public static void SANYmain(String args[]) {
+    if (args.length == 0) {
+      printUsage();
+      System.exit(-1);
```

Comment:
In keeping with the (admittedly, bad) precedent of always returning a zero exit code unless `-error-codes` is given, perhaps we should just do a `return` here to exit with 0.